Sources/DocCDocumentation/DocCDocumentation.docc/V5_API_Migration_guide.md

```diff
@@ -63,10 +61,11 @@ _ = try await Purchases.shared.recordPurchase(result)
 
 #### Observing Purchases Completed by Your App with StoreKit 1
 
-If purchases are completed by your app using StoreKit 1, you will need to explicitly configure the SDK to use StoreKit 1:
+If purchases are performed and completed by your app using StoreKit 1, outside of the RevenueCat SDK, you will need to explicitly configure the SDK to use StoreKit 1:
```

Comment:
`outside of the RevenueCat SDK`, I _think_ it makes sense to us, but I wonder if an developer who's just coming to the app would understand that you can use RevenueCat SDK without using the direct purchase methods provided by the SDK. Some may think using observer mode as being inside the RevenueCat SDK because you're still calling the SDK. 

I don't know if including (`previously known as the observer mode`) would help differentiate or just make it more confusing.

Reply:
I think I would say something like

> If you're using RevenueCat only to track purchases, and you have your own implementation of StoreKit 1 to make purchases, you will need to explicitly configure the SDK to use StoreKit 1

Reply:
I like this!